### PR TITLE
📝 add zenodo doi

### PR DESCRIPTION
**Pull request by dyollb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
+[![DOI](https://zenodo.org/badge/566711963.svg)](https://zenodo.org/badge/latestdoi/566711963)
 [![downloads](https://img.shields.io/github/downloads/ITISFoundation/IXI025/total.svg)](https://github.com/ITISFoundation/IXI025/releases)
 
+[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
```
Comment (pcrespov) [inline README.md:4]: are you sure the licence button is correct?
 why dont' you use https://shields.io/??
image

File: CITATION.cff [hunk 2]
```diff
@@ -14,6 +14,6 @@ date-released: "2022-11-16"
 identifiers:
   - description: This is the archived snapshot of version 1.0.0 of IXI025
     type: doi
-    value: "10.5281/zenodo.6563519"
+    value: "10.5281/zenodo.7331083"
```
Comment (pcrespov) [inline CITATION.cff:17]: so this reference is https://zenodo.org/record/7331083

**Review by pcrespov — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dyollb) [reply README.md:4]: image

File: CITATION.cff (hunk 2 above)
Comment (dyollb) [reply CITATION.cff:17]: yes

File: README.md (hunk 1 above)
Comment (dyollb) [reply README.md:4]: Its what the badge looks like (ugly I know): https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba

**Pull request merged**